index.html

```diff
@@ -858,10 +858,12 @@ <h2 class="heading">Modules &amp; Blueprints</h2>
 <script type="application/javascript" src="https://json2jsonp.com/?url=https://opencollective.com/generator-jhipster/tiers/bronze-sponsor/all.json?v=2&callback=ocBronzeJsonpCallback"></script>
 <script type="application/javascript" src="https://json2jsonp.com/?url=https://opencollective.com/generator-jhipster/tiers/silver-sponsor/all.json&callback=ocSilverJsonpCallback"></script>
 <script>
-    // dead simple random select of hipster family member
-    var random = Math.floor(Math.random() * (3 - 0 + 1)) + 0;
-    document.getElementById("hipster").src = "images/logo/jhipster_family_member_" + random + ".svg";
-    document.getElementById("hipster-head").src = "images/logo/jhipster_family_member_" + random + "_head.svg";
+    familyIDFromQueryParam = (new URLSearchParams(window.location.search)).get('family')
+    // simple random select of hipster family member if not specified in the `family` query param
+    var familyMemberID = familyIDFromQueryParam || Math.floor(Math.random() * (8 - 0 + 1)) + 0;
```

Comment:
why the - 0 and + 0 isn't it same as `Math.floor(Math.random() * 9)` or `Math.floor(Math.random() * (8  + 1))`?

Reply:
I didn't touch this code, I guess it's to make it explicit that we take a number between 0 and 8.